pages.fa/common/diff.md

```diff
@@ -0,0 +1,32 @@
+# diff
+
+> مقایسه فایل(ها) و پوشه(ها).
+> اطلاعات بیشتر: <https://man7.org/linux/man-pages/man1/diff.1.html>.
+
+- مقایسه فایل ها (فهرست تغییرات فایل های قدیمی به جدید) :
+
+`diff {{فایل قدیمی}} {{فایل_جدید}}`
+
+- مقایسه فایل ها، با صرف نظر از فاصله های خالی
+
+`diff --ignore-all-space {{فایل قدیمی}} {{فایل_جدید}}`
+
+- مقایسه فایل ها، با نمایش تفاوت ها در کنار هم
+
+`diff --side-by-side {{فایل قدیمی}} {{فایل_جدید}}`
+
+- مقایسه فایل ها، به نمایش تفاوت ها به صورت یکپارچه (همانند `git diff`) :
+
+`diff --unified {{فایل قدیمی}} {{فایل_جدید}}`
+
+- مقایسه بازگشتی پوشه ها (نمایش اسامی متفاوت فایل ها و پوشه ها و همچنین تغییرات فایل ها)
+
+`diff --recursive {{پوشه_قدیمی}} {{پوشه_جدید}}`
+
+- نمایش نام فایل های متفاوت مقایسه شده
+
+`diff --recursive --brief {{پوشه_قدیمی}} {{پوشه_جدید}}`
+
+- Create a patch file for Git from the differences of two text files, treating nonexistent files as empty:
```

Comment:
Seems like this line wasn't translated, can you translate it?

Reply:
Sure.